main/src/cgeo/geocaching/utils/EmojiUtils.java

```diff
@@ -90,7 +90,8 @@
             /* warning */       0x26a0, 0x26d4, 0x1f6ab, 0x1f6b3, 0x1f6d1, 0x2622,
             /* av-symbol */     0x1f505, 0x1f506,
             /* other-symbol */  0x2b55, 0x2705, 0x2611, 0x2714, 0x2716, 0x2795, 0x2796, 0x274c, 0x274e, 0x2733, 0x2734, 0x2747, 0x203c, 0x2049, 0x2753, 0x2757,
-            /* flags */         0x1f3c1, 0x1f6a9, 0x1f3f4, 0x1f3f3
+            /* flags */         0x1f3c1, 0x1f6a9, 0x1f3f4, 0x1f3f3,
+            /* numbers */       0x31, 0x32, 0x33
```

Comment:
This is some different change - introducing numbers as emojis?
Should be a separate PR, and (IIRC) should go from 0 to 9 to cover ten stage lab adventures as well.